parallel-install/example/example.go

```diff
@@ -4,7 +4,8 @@ import (
 	"context"
 	"flag"
 	"fmt"
-	"log"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/logger"
+	"go.uber.org/zap"
```

Comment:
Is "zap" a good choice? See: [Note that zap only supports the two most recent minor versions of Go.](https://github.com/uber-go/zap)

Reply:
Please bring that point to the logging-working group during the SIG quality group. We should not discuss it in this PR review :)

Reply:
It was picked as the suggestion to be used came from PoC.